Celeste.Mod.mm/Mod/Entities/EntityCollider.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Monocle;
+using System;
+using Microsoft.Xna.Framework;
+
+namespace Celeste.Mod.Entities {
+    [Tracked(false)]
+    public class EntityCollider<T> : Component where T : Entity {
+        public readonly string entityType = typeof(T).Name;
+
+        public Action<T> OnEntityAction;
+
+        public Collider Collider;
+
+        public EntityCollider(Action<T> onEntityAction, Collider collider)
```

Comment:
Might want to proivde a default `Collider collider = null`, since that's what then uses the entity collider instead of a cusotm one.

Reply:
Forgot about that, will do